### DISCUSSION TITLE
Using upload key with Bubblewrap

### DISCUSSION THREAD

**Discussion by uluhonolulu** [GoogleChromeLabs/bubblewrap#828] [Q&A] [answered]
I'm trying to update my app that I haven't touched since 2021. Play Store wouldn't accept my new AAB (nor APK) file complaining that it is signed with the wrong key.

My next idea is to request a new upload key, but how do I use it with Bubblewrap? Do I just replace `android.keystore` with the new file?

What is the best way to proceed here?

**Comment by andreban** [comment, marked as answer]
Yes, you should be able to change `android.keystore` to the path for the new key, as long as it is in the correct format.
